src/data/waypoint-landing.json

```diff
@@ -41,32 +41,6 @@
         "waypoint/configuration"
       ]
     },
-    {
-      "type": "linked_cards",
-      "cards": [
-        {
-          "heading": "Dolor sit amet",
-          "body": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incident this text is written so that it's very very long but it should be truncated in the card to a max of three lines to match designs",
-          "url": "https://www.waypointproject.io"
-        },
-        {
-          "heading": "Dolor sit amet",
-          "body": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incident this text is written so that it's very very long but it should be truncated in the card to a max of three lines to match designs",
-          "url": "https://www.waypointproject.io"
-        },
-        {
-          "heading": "Dolor sit amet",
-          "body": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incident this text is written so that it's very very long but it should be truncated in the card to a max of three lines to match designs",
-          "url": "https://www.waypointproject.io"
-        }
-      ]
-    },
-    {
-      "type": "heading",
-      "heading": "Case Studies",
-      "level": 2,
-      "size": 400
-    },
     {
       "type": "linked_cards",
       "cards": [
```

Comment:
@topfunky right below is some more lorem ipsum ([Preview Link 🔍](https://dev-portal-git-ggwaypoint-landing-hashicorp.vercel.app/waypoint)).  Do we want to also remove that?

![CleanShot 2022-05-17 at 13 05 35@2x](https://user-images.githubusercontent.com/2105067/168900510-bfa0cb80-368f-43ea-86cf-eec7b4f99bd2.png)